### PR TITLE
materia.json: annotate with more information

### PR DESCRIPTION
**Pull request by cpankow**
Addresses part of #42 .

### CODE REVIEW

**Comment by cpankow** [comment]
Hmmm... I'm not sure I understand the test failures. Let me know if you understand what needs to be changed.

File: projects/shared/src/lib/materia.json [hunk 1]
```diff
@@ -1,338 +1,791 @@
...
-    },
-    {
-        "name": "Quadra Magic",
-        "ty": "support"
-    },
-    {
-        "name": "Sneak Attack",
-        "ty": "support"
+        "clr": "Green",
+        "ty": "Magic",
```
Comment (konkers) [inline projects/shared/src/lib/materia.json]: These should be lower case so they match: https://github.com/konkers/ff7run/blob/fdf46bfad2116f37e5934ab1a43bed31584cfd35/projects/shared/src/lib/model.ts#L3

File: projects/shared/src/lib/materia.json [hunk 2]
```diff
@@ -1,338 +1,791 @@
...
-        "ty": "support"
-    },
-    {
-        "name": "Quadra Magic",
-        "ty": "support"
-    },
-    {
-        "name": "Sneak Attack",
-        "ty": "support"
+        "clr": "Green",
```
Comment (konkers) [inline projects/shared/src/lib/materia.json]: Color seems redundant with type.  Is there a compelling reason to include it?  I've been keying color off of type here: https://github.com/konkers/ff7run/blob/fdf46bfad2116f37e5934ab1a43bed31584cfd35/src/app/materia/materia.component.scss

File: projects/shared/src/lib/materia.json [hunk 3]
```diff
@@ -1,338 +1,791 @@
...
+        "ty": "Magic",
+        "eff": [
```
Comment (konkers) [inline projects/shared/src/lib/materia.json]: I'd like to keep fields as full words when possible.  "type" is "ty" as it conflicts with keywords in many languages.

File: projects/shared/src/lib/materia.json (hunk 1 above)
Comment (cpankow) [reply projects/shared/src/lib/materia.json]: Fixed.

File: projects/shared/src/lib/materia.json (hunk 2 above)
Comment (cpankow) [reply projects/shared/src/lib/materia.json]: Just for completeness. The latest commit dropped this.

File: projects/shared/src/lib/materia.json (hunk 3 above)
Comment (cpankow) [reply projects/shared/src/lib/materia.json]: Ah, I thought you were prioritizing brevity and purposefully had used shorter names. That's reverted back to full name snow.

**Comment by konkers** [comment]
Do you mind squashing these commits into on?  I like to use "rebase and merge" to keep a clean history.

**Comment by konkers** [comment]
Superceded by #46

**Pull request closed without merging**